tests/Plugin.php

```diff
@@ -1,7 +1,9 @@
 <?php
 
-use Pest\PluginName\Plugin;
+it('has global function', function () {
+    assertTrue(function_exists('livewire'));
+});
 
-it('has plugin', function () {
-    assertTrue(class_exists(Plugin::class));
+it('has plugin trait method', function () {
```

Comment:
You should call the method using the variants that the mthod have an assert a few things.

Reply:
Just working on this. 👍